epictrack-api/README.md

```diff
@@ -7,31 +7,118 @@
 EAO Project report services.
 
 
-## Development Setup
+# EpicTrack Local Setup
+
+## GitHub Repository
+[EPIC.track GitHub](https://github.com/bcgov/EPIC.track)
+- Fork a new branch and point to the `develop` branch.
+
+## Required Access
+Zenhub, Openshift, Keycloak
+- You will need access. For now, contact dinesh.pb@aot-technologies.com
+
+## Development Environment Setup
+
+### Windows
+
+#### Database
+- Set up a Postgres database locally: [Docker Postgres](https://hub.docker.com/_/postgres)
+- If you don’t have Docker, install it from: [Docker Desktop Install](https://docs.docker.com/desktop/install/windows-install/)
+
+### API Setup
+
+1. Open a terminal and navigate to `epictrack-api`
+2. Run `docker-compose up`
+3. In a separate terminal, navigate to `reports-api`
```

Comment:
we don't have reports_api folder. It is just api now

Reply:
just updated it